src/lib.rs

```diff
@@ -10,24 +10,34 @@ pub mod merkle;
 // RE-EXPORTS
 // ================================================================================================
 
+pub use winter_crypto::{Digest, ElementHasher, Hasher};
 pub use winter_math::{fields::f64::BaseElement as Felt, FieldElement, StarkField};
 
 pub mod utils {
     pub use winter_utils::{
         collections, string, uninit_vector, ByteReader, ByteWriter, Deserializable,
         DeserializationError, Serializable, SliceReader,
     };
+
+    #[cfg(not(feature = "std"))]
+    pub use alloc::borrow::Cow;
+
+    #[cfg(feature = "std")]
+    pub use std::borrow::Cow;
```

Comment:
It seems like we may not need these ones any more (at least for now).

Reply:
True, fixed!